### ISSUE TITLE
Pretrained model for bulk low-quality data

### ISSUE THREAD

**Issue by avantikalal** [clara-parabricks/AtacWorks#236]
Hi @avantikalal, in cases where noisy samples have coverage >= that of the clean samples, should users _always_  forego training and use your pretrained model, nvidia:atac_bulk_lowqual_20m_20m? 

For example:
![image](https://user-images.githubusercontent.com/44478133/113457216-3b7c7600-93dd-11eb-9a35-212a9171ec4d.png)

If the pretrained model is not successful in reducing noise, are there training parameters that should be considered when constructing a custom model.

Thanks for the help—atacworks looks like a gamechanger!

_Originally posted by @umasstr in https://github.com/clara-parabricks/AtacWorks/issues/221#issuecomment-812733148_

**Comment by avantikalal** [comment]
Hi @umassstr, for the best results we recommend training your own model, if you have matched low/high quality data available. The noisy data and clean data could have any coverage, what is important is that the noisy data used for training should have similar coverage to the noisy data to which you intend to apply the model.

**Comment by umasstr** [comment]
Thanks for following up. In agreement with your above post, I found the pretrained model to be incompatible with my dataset and some sample ENCODE datasets. The custom trained model worked well, but there's a constant, low-level background in the output (green below). 

Purple: original (macs)
Green: custom model
Orange: atac_bulk_lowqual_20m_20m

![image](https://user-images.githubusercontent.com/44478133/113596082-d0fb4e00-9607-11eb-857e-2ca7c09101ab.png)

For reasons unknown, it looks like your pipeline needs a non-zero integer at all coordinates. >25M peaks have a 1.0 signal, comprising the majority of the bedgraph. 

`$ cut -f4 2296_infer.track.bedGraph | sort | uniq -c | head -1` 
 
>26356100 1.0

Unfortunately, simply subtracting 1 from all coordinates doesn't produce an ideal track, though it could be useful for analytical purposes. Any idea how I can get rid of this artifact produced by custom models? I can file a new issue if you'd like.

**Comment by avantikalal** [comment]
Great to see the custom model working well overall, though I agree the low-level background is a problem. The model shouldn't require a nonzero output at every coordinate. Could you share the config file generated by your model training run?

**Comment by umasstr** [comment]
Indeed, the output is pretty impressive, especially if I set the track bounds to hide the 1.0 background:

![image](https://user-images.githubusercontent.com/44478133/113602908-41f33380-9611-11eb-9c38-90901e7765f5.png)

Here is a link to the training [config](https://www.dropbox.com/sh/bm49m534jqiusrf/AADzNLbQHf-PbuDauYyhWjZaa)

**Comment by avantikalal** [comment]
One thing that might help is to train your model using the Poisson loss function for regression instead of the MSE/Pearson loss functions. This has given us better regression results and will become the default regression loss in the future. 

To train a model using Poisson loss, you can set `--mse_loss` equal to 0, `--pearson_loss` equal to 0, and `--poisson_loss` equal to 1. 

If you do try this, please let us know if it gives better results!

**Comment by umasstr** [comment]
Hi @avantikalal, I tried the parameters above, but the results did not look good. 

![image](https://user-images.githubusercontent.com/44478133/117913078-b1161480-b2ae-11eb-9ecd-f5917886201b.png)

I am a little concerned about the fact that, for each noisy BW, I have to retrain a model. None of the prebuilt models work with my data and none of my samples' models work on the others.
